Lib/test/test_re.py

```diff
@@ -700,6 +700,39 @@ def test_other_escapes(self):
             with self.subTest(c):
                 self.assertRaises(re.error, re.compile, '[\\%c]' % c)
 
+    def test_named_unicode_escapes(self):
+        # test individual Unicode named escapes
+        suites = [
+            [   # basic matches
+                ['\u2014', r'\u2014', '\N{EM DASH}',
+                 r'\N{EM DASH}'],                               # pattern
+                ['\u2014', '\N{EM DASH}', '—', '—and more'],    # matches
```

Comment:
It is hard to see differences between different dashes on terminal. Use just `\u2014` or `\N{EM DASH}`.